blueretro_presets.js

```diff
@@ -414,5 +449,41 @@ function btConn() {
 }
 
 function selectInput() {
-    document.getElementById("desc").textContent = presets[Number(this.value)].desc;
+    //only change the description if selected input is not the placeholder
+    if(Number(document.getElementById("presetsName").value) == -1 || Number(document.getElementById("consoleName").value) == -1)
```

Comment:
Sorry to be picky about style, but make sure you got a space between the if (or any other statement) and the parentheses/braces. Same for curly bracket that open following the closing parenthesis.

Reply:
double checked and changed all i could find to match the style. if you catch any more let me know!